snap/snapcraft.yaml

```
@@ -12,6 +12,9 @@ grade: stable
 confinement: strict
 base: core18
 
+system-usernames:
+  snap_microk8s: shared
```

you can make this work without the snapcraft change using passthrough

```suggestion
passthrough:
  system-usernames:
    snap_microk8s: shared
```